src/Fixer/Strict/DeclareStrictTypesFixer.php

```diff
@@ -63,6 +88,16 @@ public function isRisky(): bool
         return true;
     }
 
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('keep_loose_types', 'Whether existing strict_types=0 should be preserved instead of upgrading to strict_types=1.'))
```

Comment:
let's be future-ready for potential config to configure 0 by default
```suggestion
            (new FixerOptionBuilder('preserve_existing_declaration', 'Whether existing strict_types=? should be preserved and not overridden.'))
```